protostar/commands/cairo1_commands/test_cairo1_command.py

```diff
@@ -108,32 +118,95 @@ async def run(self, args: Namespace) -> TestingSummary:
             args.json = None
         messenger = self._messenger_factory.from_args(args)
         cache = TestCommandCache(CacheIO(self._project_root_path))
-        test_command = TestCommand(
-            project_root_path=self._project_root_path,
-            protostar_directory=self._protostar_directory,
-            project_cairo_path_builder=self._project_cairo_path_builder,
-            log_color_provider=self._log_color_provider,
-            cwd=self._cwd,
-            active_profile_name=self._active_profile_name,
-            messenger_factory=self._messenger_factory,
-        )
-        summary = await test_command.test(
+
+        summary = await self.test(
             targets=cache.obtain_targets(args.target, args.last_failed),
             ignored_targets=args.ignore,
             cairo_path=args.linked_libraries,
-            disable_hint_validation=False,
-            profiling=False,
             no_progress_bar=args.no_progress_bar,
-            safe_collecting=False,
             exit_first=args.exit_first,
-            seed=None,
-            max_steps=None,
             slowest_tests_to_report_count=args.report_slowest_tests,
-            gas_estimation_enabled=False,
             messenger=messenger,
-            use_cairo1_test_runner=True,
         )
         cache.write_failed_tests_to_cache(summary)
 
         summary.assert_all_passed()
         return summary
+
+    async def test(
+        self,
+        targets: List[str],
+        messenger: Messenger,
+        ignored_targets: Optional[List[str]] = None,
+        cairo_path: Optional[List[Path]] = None,
+        no_progress_bar: bool = False,
+        exit_first: bool = False,
+        slowest_tests_to_report_count: int = 0,
+    ) -> TestingSummary:
+        include_paths = [
+            str(path)
+            for path in self._project_cairo_path_builder.build_project_cairo_path_list(
+                cairo_path or []
+            )
+        ]
+        include_paths.append(
+            str(self._protostar_directory.protostar_cairo1_corelib_path)
+        )
```

Comment:
[nitpick]: If it's always appended it doesn't make sense to have it in a seprarate line